native/src/main/java/io/ballerina/stdlib/serdes/protobuf/ProtobufMessageFieldBuilder.java

```diff
@@ -77,8 +77,8 @@ public int getFieldNumber() {
 
     public String toString(String indentation) {
         String fieldLabel = getFieldLabel().equals(REPEATED_LABEL) ? REPEATED_LABEL + SPACE : EMPTY_STRING;
-        return indentation + SPACE + fieldLabel + getFieldType() + SPACE + getFieldName() + SPACE
-                + " = " + getFieldNumber() + ";\n";
+        return indentation + SPACE + fieldLabel + getFieldType() + SPACE + getFieldName() + SPACE + " = "
+                + getFieldNumber() + ";\n";
```

Comment:
Shall we use a string builder here?

Reply:
Since there is no looping involved using string builder is not beneficial here right ?
IDE also suggest to use string instead of string builder here.